### DISCUSSION TITLE
timed out waiting for the condition

### DISCUSSION THREAD

**Discussion by bwolmarans** [sse-secure-systems/connaisseur#504] [Q&A]
Hi, how may I find logs or troubleshoot this one?  I was able to install a different admission controller (sysdig) successfully, and I then uninstalled sysdig adminssions controller.  

Then I tried to install this one but I got:

:~/connaisseur$ helm install connaisseur helm --atomic --create-namespace --namespace connaisseur
Error: INSTALLATION FAILED: release connaisseur failed, and has been uninstalled due to atomic being set: timed out waiting for the condition

**Comment by xopham** [comment]
Hi @bwolmarans :wave: given that you seem to encounter problems during installation, I would recommend to check the `kubectl describe` events of deployment and replicaset and potentially pods (if those ever start) to see why they don't come up.
Let us know if that helps or what comes up :pray:
